src/Init/Data/String/Extra.lean

```diff
@@ -98,4 +98,32 @@ def removeLeadingSpaces (s : String) : String :=
   let n := findLeadingSpacesSize s
   if n == 0 then s else removeNumLeadingSpaces n s
 
+/--
```

Comment:
Some tests for `crlfToLf` would be nice

Reply:
Note that this is not a new function, it is moving from lake, where it has been involved in hash calculations for some time. So we have some evidence that this works to normalize e.g. the files in mathlib when checking them out on windows / linux, because otherwise `lake exe cache` would break.

Reply:
Still, I added some tests to make sure that it does what we think it does in some corner cases.